steps/synopsys-dc-synthesis/configure.yml

```diff
@@ -51,6 +51,21 @@ parameters:
   # This is useful for hierarchical LVS when multiple blocks use modules
   # with the same name but different definitions.
   uniquify_with_design_name: True
+  #Enable supression of selected warnings/messages. 
+  suppress_msg: True
+  #List of suppressed messages. Select carefully
```

Comment:
This looks useful! Thanks for the contribution!

Given the careful selection, maybe you can default this to False though... and set it to True in your own graph?

Also very minor, but can you add a space between "# Enable" and "# List" too?

Looks great!

Reply:
Yes, sure

Reply:
I've changed it as suggested